src/components/utils/DEPagination.js

```diff
@@ -0,0 +1,167 @@
+import React from "react";
+import { makeStyles, useTheme } from "@material-ui/core/styles";
+import Pagination from "@material-ui/lab/Pagination";
+import {
+    Button,
+    ClickAwayListener,
+    Grid,
+    Grow,
+    Hidden,
+    MenuItem,
+    MenuList,
+    Paper,
+    Popper,
+    Tooltip,
+    useMediaQuery,
+} from "@material-ui/core";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+import { injectIntl } from "react-intl";
+import { build, formatMessage, withI18N } from "@cyverse-de/ui-lib";
+import ids from "./ids";
+import intlData from "./messages";
+
+const useStyles = makeStyles((theme) => ({
+    paper: {
+        flexGrow: 1,
```

Comment:
Is there something else we can do to center the pagination? Maybe this is a spot where [Container](https://material-ui.com/components/container/) might be useful?  At least in the screenshots it looks like it's always in the same spot, so it looks more centered if you have a lot of pages, but looks offset if you only have a few pages. And then on mobile, it's all the way to the left with a gap on the right.

Reply:
There is not much choice when it comes mobile. We have to align left otherwise even with couple of pages, it will overflow into 2nd line. But I will try `Container` and `Box` to see if it better suited for aligning stuff to the center in large screens.